docs/source/components/pipeline.rst

```diff
@@ -45,6 +45,31 @@ The reason behind this is that OpenVINO doesn't provide version inside the blob.
   # Set the correct version:
   pipeline.setOpenVINOVersion(depthai.OpenVINO.Version.VERSION_2021_4)
 
+Specifying arguments for :code:`getOutputQueue` method
+######################################################
+
+When obtaining the output queue (example code below), the :code:`maxSize` and :code:`blocking` arguments should be set depending on how
+the messages are intended to be used, where :code:`name` is the name of the outputting stream.
+
+.. code-block:: python
+
+  with dai.Device(pipeline) as device:
+    queueLeft = device.getOutputQueue(name="manip_left", maxSize=8, blocking=False)
+    
+If only the latest results are relevant and previous don't matter, one can set :code:`maxSize = 1` and :code:`blocking = False`.
```

Comment:
Maybe here we should also mention that if host isn't able to `get` all messages from the queue fast enough (eg. one-threaded environment that does some heavy computing), you should set the `maxSize` to higher number. If you are sure you will be able to `get` msg from queue faster than eg 33ms (if msgs come at 30FPS), than you can leave it at maxSize=1.